weaviate/collections/classes/config.py

```diff
@@ -684,23 +691,31 @@ def palm(
 
     @staticmethod
     def aws(
-        model: str,
         region: str,
+        service: Union[AWSService, str] = "bedrock",
+        model: Optional[str] = None,
```

Comment:
this is sadly a breaking change, because of the order :(

Reply:
if people have positional arguments in there, it would break

Reply:
ah bummer - if we need to break anyways, let's add `*` as a first parameter then all parameters are kwargs

Reply:
[Found this](https://github.com/weaviate/weaviate-python-client/blob/9480b60ca3c3ae66df9aaf52a09bffcf5cbc600b/weaviate/collections/classes/config_vectorizers.py#L547) while looking for the right place to edit. So maybe we could follow the style.

Reply:
I see ok - I would still be fine with changing it kwargs-only